cora_description/models/base/model.config

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<model>
+  <name>_</name>
+  <version>1.0</version>
+  <sdf version="1.6">model.sdf</sdf>
+
+  <author>
+    <name>Rafi Abdullah</name>
+    <email>rafi@openrobotics.org</email>
+  </author>
+
+  <description>
+    Fake model to prevent errors from GAZEBO_MODEL_PATH
```

Comment:
I think we can add a proper description here.

Reply:
35cc281